anago

```diff
@@ -952,17 +952,21 @@ for label in ${!RELEASE_VERSION[@]}; do
   common::stepheader "PUSH ${RELEASE_VERSION[$label]} IMAGES"
   ##############################################################################
   common::runstep release::gcs::copy_release_artifacts \
```

Comment:
minor API nit: it's a little unfortunate that `copy_release_artifacts` uses `TYPE/VERSION` as one argument whereas `publish_version` uses `TYPE VERSION` as two separate arguments.